pydex/core/oa/manager.py

```diff
@@ -82,9 +82,9 @@ def solve(self, n_exp, y0, atol=None, rtol=1e-3, assess_potential_oa_gain=False,
         while not self.converged:
             """ Primal problem: log_det evaluation only """
             if self.iteration_no == 0:
-                self.primal_problem.efforts = y0
```

Comment:
will the kwarg "y0" be used if self is added on this line?

Reply:
It is used to initialise the value of self.y0 (line 71) that would otherwise be None.